scripts/import-sdvx.js

```diff
@@ -91,7 +117,51 @@ async function main() {
   );
 }
 
-function buildSong(song) {
+function determineDiffClass(song, chartType) {
+  if (chartType !== "infinite") {
+    return chartType;
+  }
+  const infVersion = parseInt(song.info[0].inf_ver[0]._);
+  switch (infVersion) {
+    case 2:
+      return "infinite";
+    case 3:
+      return "gravity";
+    case 4:
+      return "heavenly";
+    case 5:
+      return "vivid";
+  }
+}
+
+function filterUnplayableSongs(song) {
+  const songsIdsToSkip = [
```

Comment:
Like `availableJackets`, this can also be a `Set` (and it can be defined just once, outside this function, so it doesn't get re-created on every call)